docnado/docnado.py

```diff
@@ -782,10 +786,18 @@ def __init__(self, md_file):
 
         self.references = set()
         for k, v in tags_to_search.items():
-            for tag in soup.find_all(k):
-                val = tag.get(v)
-                if val:
-                    self.references.add(val)
+            links = soup.find_all(k)
+
+            for link in links:
+                if link.get('href'):
+                    if link.get('href').find('http:') > -1 or link.get('href').find('https:') > -1:
```

Comment:
what is this doing?

Reply:
Finding links with the protocol "http" or "https".